content/influxdb/cloud-iox/sql-reference.md

```diff
@@ -0,0 +1,614 @@
+---
+title: InfluxDB SQL reference
+description: >
+  InfluxDB SQL reference
+menu:
+  influxdb_cloud_iox:
+    name: InfluxDB SQL reference
+    parent: Query data with SQL
+weight: 190
+---
+
+InfluxDB Cloud backed by InfluxDB IOx uses the Apache Arrow DataFusion implementation of SQL.  
```

Comment:
I don't know if you have linked it elsewhere but it might help to link  https://arrow.apache.org/datafusion/ here for context

Reply:
added link